deployments/infrastructure-env-production.nix

```diff
@@ -37,12 +37,28 @@ in {
     deployment.ec2.elasticIPv4 = resources.elasticIPs.cardanod-ip;
   };
 
+  kite = { config, pkgs, resources, ... }: {
```

Comment:
This will have to be part of each `hydra-slave` for proper integration, as the agents will run there.

Reply:
Understood..

Reply:
This item is now tracked in https://iohk.myjetbrains.com/youtrack/issue/DEVOPS-527